### PR TITLE
add the pruned_transducer_stateless7_streaming recipe for commonvoice

### PR DESCRIPTION
**Pull request by manbaaaa**
This PR adds the pruned_transducer_stateless7_streaming recipe for the CommonVoice dataset, and also updates isort from version 5.10.1 to 5.12.0 because I encountered an error during the code style check. I resolved the issue by updating the version.

[INFO] Installing environment for https://github.com/pycqa/isort.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/home/lishaojie/.cache/pre-commit/repoy5j5vfwy/py_env-python3/bin/python', '-mpip', 'install', '.')
return code: 1
stdout:
    Processing /home/lishaojie/.cache/pre-commit/repoy5j5vfwy
      Installing build dependencies: started
      Installing build dependencies: finished with status 'done'
      Getting requirements to build wheel: started
      Getting requirements to build wheel: finished with status 'done'
      Preparing metadata (pyproject.toml): started
      Preparing metadata (pyproject.toml): finished with status 'error'
    
stderr:
      error: subprocess-exited-with-error
      
      × Preparing metadata (pyproject.toml) did not run successfully.
      │ exit code: 1
      ╰─> [14 lines of output]
          Traceback (most recent call last):
            File "/home/lishaojie/.cache/pre-commit/repoy5j5vfwy/py_env-python3/lib/python3.9/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
              main()
            File "/home/lishaojie/.cache/pre-commit/repoy5j5vfwy/py_env-python3/lib/python3.9/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
              json_out['return_val'] = hook(**hook_input['kwargs'])
            File "/home/lishaojie/.cache/pre-commit/repoy5j5vfwy/py_env-python3/lib/python3.9/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 149, in prepare_metadata_for_build_wheel
              return hook(metadata_directory, config_settings)
            File "/tmp/pip-build-env-rlm5usec/overlay/lib/python3.9/site-packages/poetry/core/masonry/api.py", line 40, in prepare_metadata_for_build_wheel
              poetry = Factory().create_poetry(Path(".").resolve(), with_groups=False)
            File "/tmp/pip-build-env-rlm5usec/overlay/lib/python3.9/site-packages/poetry/core/factory.py", line 57, in create_poetry
              raise RuntimeError("The Poetry configuration is invalid:\n" + message)
          RuntimeError: The Poetry configuration is invalid:
            - [extras.pipfile_deprecated_finder.2] 'pip-shims<=0.3.4' does not match '^[a-zA-Z-_.0-9]+$'
          
          [end of output]
      
      note: This error originates from a subprocess, and is likely not a problem with pip.
    error: metadata-generation-failed
    
    × Encountered error while generating package metadata.
    ╰─> See above for output.
    
    note: This is an issue with the package mentioned above, not pip.
    hint: See above for details.
    
Check the log at /home/lishaojie/.cache/pre-commit/pre-commit.log

### CODE REVIEW

File: egs/commonvoice/ASR/local/compile_hlg.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
```
Comment (csukuangfj) [inline egs/commonvoice/ASR/local/compile_hlg.py:1]: Could you replace it with a symlink?

Comment (manbaaaa) [reply egs/commonvoice/ASR/local/compile_hlg.py]: ok

**Comment by csukuangfj** [comment]
@manbaaaa 

could you update RESULTS.md to include WERs and links to pre-trained models?

**Comment by manbaaaa** [comment]
@csukuangfj ok, I have already added it.

**Comment by csukuangfj** [comment]
@manbaaaa 

Sorry that we missed this pull-request. Could you please fix the conflicts?

**Pull request merged**